imagetest/cmd/manager/main.go

```diff
@@ -158,9 +160,25 @@ func main() {
 		return
 	}
 
-	out, err := imagetest.RunTests(ctx, testWorkflows, *outPath, *project, *zone, *parallelCount)
+	suites, err := imagetest.RunTests(ctx, testWorkflows, *project, *zone, *parallelCount)
 	if err != nil {
 		log.Fatalf("Failed to run tests: %v", err)
 	}
-	fmt.Printf("%s\n", out)
+	if suites.Errors != 0 || suites.Failures != 0 {
+		log.Fatalf("test suite has error or failure")
```

Comment:
this will cause it to stop, and not display or write the junit output. i think we should log this but not with fatal, and then we should exit with an appropriate exit code at the end

Reply:
Oh. right, I think we could just simple move the `fmt.Printf("%s\n", out)` ahead of this.